src/Config/DataApiBuilderException.cs

```diff
@@ -16,6 +16,7 @@ public class DataApiBuilderException : Exception
         public const string CONNECTION_STRING_ERROR_MESSAGE = "A valid Connection String should be provided.";
         public const string GRAPHQL_FILTER_ENTITY_AUTHZ_FAILURE = "Access forbidden to the target entity described in the filter.";
         public const string GRAPHQL_FILTER_FIELD_AUTHZ_FAILURE = "Access forbidden to a field referenced in the filter.";
+        public const string AUTHORIZATION_FAILURE = "Authorization Failure: Access Not Allowed.";
```

Comment:
can we have a similar message like: "Access forbidden to ..."

Reply:
Can you elaborate a bit here?

Reply:
The other 2 AuthZ failure messages start with "Access forbidden to"  as seen on the two lines above. Can we have a similar failure message?

Reply:
Added a substatuscode to indicate db policy failure. Since its not the case when access to some field or entity is prohibited but instead a case when db policy is not satisfied.

Reply:
database policies are part of authorization permissions.